spec/metric_fu/metrics/reek/generator_spec.rb

```diff
@@ -99,9 +100,12 @@
                             smell_type: "Duplication",
                             lines: [6, 9])
         ]
+        @examiners = [
+          instance_double(Reek::Examiner, smells: @smells)
+        ]
         @lines = instance_double(@examiner, smells: @smells)
```

Comment:
@etagwerker is it necessary to keep this variable? I noticed that you removed it in the line below

Reply:
Probably not. I'll review this today.